### ISSUE TITLE
ALL: Separate clients into IOClient and ManagementClient

### ISSUE THREAD

**Issue by fsanaulla** [fsanaulla/chronicler#66]
It's will be nice to have the option to choose functional of the client. The most common case is simply IO, the solutions create several methods in Influx object. It's, should look like that:
```
def createIOClient(...): IOClient // only IO operations
def createManagementClient(...): ManagemetnClient // influx management operations
def create(...): Client // combination of IO and Management clients
```

**Comment by fsanaulla** [comment]
Related to #75.

**Comment by fsanaulla** [comment]
Solved in #81.
